src/quo2/components/avatars/channel_avatar/style.cljs

```diff
@@ -5,14 +5,24 @@
 
 (defn outer-container
   [{:keys [size color]}]
-  (let [size (if (= size :size/l) 32 24)]
-    {:width            size
-     :height           size
-     :border-radius    size
+  (let [container-size (case size
+                         :size/l-64 64
```

Comment:
let's use `:size-64` so 👍 
see discussion and result here
https://github.com/status-im/status-mobile/pull/17279

could you also adjust `:size/l` -> :size-32

Reply:
Oh yes, I read  the discussion after having this PR ready. I will fix it across these components 👍

Thank you!